.gitignore

```diff
@@ -3,3 +3,5 @@
 .DS_Store
 .envrc
 .vscode
+pipelines/
+defaults/
```

Comment:
why did you add these?

Reply:
I usually keep these around for testing. felt a bit anxious to accidentally commit some of it, so I added it to ignore to be safe